Automattic/wp-calypso
# Fix/legace fse rest routes

#### Changes proposed in this Pull Request

* ensure that the REST API serves our controller

#### Testing instructions

* apply patch, ensure that template parts load in the page editor
* with REST API sandboxed
